lib/migration.js

```diff
@@ -260,4 +239,35 @@ module.exports = function(IBMDB) {
       return false;
     }
   };
+  IBMDB.prototype.alterTable = function(model, actualFields, actualIndexes,
+  callback, checkOnly) {
+    var self = this;
+    var sql = [];
+    var tasks = [];
+    self.changeOrAddColumns(model, actualFields, function(res) {
```

Comment:
The consistent name in other connectors is `getAddModifyColumns`, please rename it.